components/calendar/src/calendar-input/calendar-input.js

```diff
@@ -38,6 +38,7 @@ export const CalendarInput = ({
     strictValidation,
     inputWidth,
     dataTest = 'dhis2-uiwidgets-calendar-inputfield',
+    pastOnly,
```

Comment:
as a general approach - shouldn't this just be controlled by `minDate` / `maxDate` from the client? The client should set the max date (we have a helper that gets the current date in the specified calendar) .. it will then avoid having hardcoded number of years that seems tracker-specific right now

Reply:
The minDate and maxDate validation parameters are not being passed through to the calendar component. Also, the useDatePicker hook is not utilizing these validation parameters because the validateDateString function is being called directly within the calendar input component.